### ISSUE TITLE
te

### ISSUE THREAD

**Issue by maghsk** [TryGhost/Ghost-Android#48]
- Ghost version: 3.4.0 
- App version: 1.2.4

----

I inputed my blog url and clicked next step and then I got stucked.
I can login to my server by chrome browser.
![image](https://user-images.githubusercontent.com/16971372/73715119-e350d200-474d-11ea-8f50-130201c059b2.png)

(I'm using an Android 10 Pad.)

**Comment by stuartwakefield** [comment]
Same here, on Pixel 3. Trying to connect to standard ghost.io address.

**Comment by stuartwakefield** [comment]
Looks like the same issue as #45. Possibly related to version difference between app and ghost API.

**Comment by argestes** [comment]
@stuartwakefield that's correct. I looked at the source code and there is no support for latest version of the API. AFAIK there is no way to enable the old api.

**Comment by JimTim** [comment]
At the moment I'm working on an Android/iOS app that connects to the Api in version 3. It is written in the programming language Flutter and in it I am a real newbie and learn something new every day. Therefore the app is still very much in the beginning and needs some iterations.  So there is hope.

I hope it is ok to mention this here. If not, feel free to delete the post.
![Screenshot_20200213-235706](https://user-images.githubusercontent.com/6737248/74486307-f9346480-4ebc-11ea-8005-839fe05aca66.png)
![Screenshot_20200213-235828](https://user-images.githubusercontent.com/6737248/74486314-fc2f5500-4ebc-11ea-9001-f31fccbbf105.png)
![Screenshot_20200213-235846](https://user-images.githubusercontent.com/6737248/74486322-fe91af00-4ebc-11ea-808f-c79f1dd6a25c.png)
![Screenshot_20200213-235900](https://user-images.githubusercontent.com/6737248/74486323-fe91af00-4ebc-11ea-8841-3a770d4c9cbf.png)
